New blog post: January EcoDev Updates (#25)
waku-org/blog.waku.org: posts/2024-02-01-january-ecodev.mdx

---
layout: post
name: 'January EcoDev Roundup'
title: 'The Waku Roadmap'
date: 2024-02-01 00:00:00
authors: [amelia]
published: true
slug: january-ecodev
categories: waku, progress
image: /img/black-waku-logo-with-name.png
hide_table_of_contents: false
---

# January EcoDev Roundup
Updates from the EcoDev team in January 2024, plus highlights from the first Waku community call. 

<!--truncate-->

The EcoDev team at Waku covers community, growth, partnerships, and communications. Each month, we will share the key highlights and progress of the team. You can [subscribe](https://waku.org/subscribe/) to the monthly newsletter if you prefer to receive an abridged version of these updates in your inbox. 

## Events and Hackathons

In 2023, Waku participated in three hackathons and eight events across four continents. We are gearing up for another year full of events; expect to see us all around the globe once again. Waku will be attending two key in-person events in Q1 2024. We will focus on growing the developer community and meeting with projects to support further integrations with Waku.

**ETHDenver 23rd February - 3rd March**

> Meet the Waku team attending ETHDenver: Aaron Bendesrky, Waku Program Manager at Logos, Pedro Lains, Business Development Lead, and Guru T, Developer Relationships Engineer. ETHDenver is a great opportunity to meet the Waku team face to face; atendees can book an appointment with the team [here](https://waku.org/events/eth-denver/).

Reviewer: ```suggestion
Meet the Waku team attending ETHDenver: Aaron Bendesrky, Waku Program Manager at Logos, Pedro Lains, Business Development Lead, and Guru T, Developer Relationships Engineer. ETHDenver is a great opportunity to meet the Waku team face to face; attendees can book an appointment with the team [here](https://waku.org/events/eth-denver/).
```

The story was also covered in [CoinTelegraph](https://cointelegraph.com/press-releases/vitalik-buterin-recognises-waku-as-the-peer-to-peer-communications-layer-of-gavin-woods-web3-vision), [CryptoDaily](https://cryptodaily.co.uk/2024/01/meet-waku-the-peer-to-peer-communications-protocol-recognized-by-vitalik-buterin), and [The Defiant](https://thedefiant.io/waku-the-p2p-communication-protocol-for-web3-recognized-by-vitalik-buterin). 

**Waku was voted as a top privacy project of 2023**: Web3 Privacy Now included Waku on their long list of [top privacy-focused projects](https://mirror.xyz/0x0f1F3DAf416B74DB3DE55Eb4D7513a80F4841073/LdOt0mlG_nXqhnmLf7s-X13NDNdpwQw72ESKaJq-0zA) of the year. The finalists were voted on by over 100 privacy enthusiasts, developers, researchers, cryptographers, cypherpunks, and community builders from DarkFi, Nym, Swarm, Logos, Railgun, and many other organisations.

> ## Technical Releases

Reviewer: There was more technical reealse than that. Could we dig in the waku wednesday updates to find more relevant?
Many @chair28980 can help extract relevant update from weekly updates (and acutally point to roadmap website))

**The Waku React package:** The package provides the components and UI adapters needed to integrate Waku SDK into React applications effortlessly. We developed the package in response to feedback from hackathon participants last year. The documentation, including detailed steps on using the package, [can be found here](https://docs.waku.org/guides/js-waku/use-waku-react/). 

There is also a [beginner-friendly tutorial](https://blog.waku.org/tictactoe-tutorial/) for building a simple tic-tac-toe game that can be played by two players who are connected using Waku.

> **Open Bounties:** We have reviewed and revised the Waku bounties, ensuring that these tasks are beginner-friendly. Our goal is to make it easier for newcomers to Waku to contribute and, in turn, support the growth of the ecosystem.Anyone can work on the bounties at their own pace, as there is no hard cut-off date. When a bounty proposal is approved, it will be marked as ‘Assigned’. Find the open bountie [here.](https://github.com/waku-org/bounties/issues)

Reviewer: ```suggestion
**Open Bounties:** We have reviewed and revised the Waku bounties, ensuring that these tasks are beginner-friendly. Our goal is to make it easier for newcomers to Waku to contribute and, in turn, support the growth of the ecosystem. Anyone can work on the bounties at their own pace, as there is no hard cut-off date. When a bounty proposal is approved, it will be marked as ‘Assigned’. Find the open bountie [here.](https://github.com/waku-org/bounties/issues)
```

## Team Updates

The team has expanded this month, with the new EcoDev Team Lead Marina and a new Business Development Lead, Pedro, joining the team.

You can connect with [Marina](https://twitter.com/Crypto_marina) and [Pedro](https://twitter.com/bigbraincap) on X.

If you are interested in working with Waku, [we are currently hiring a Software Engineer](https://boards.greenhouse.io/waku/jobs/5607301) to work with the Waku Chat SDK. The position is fully remote, and we hire globally. 

---

## Waku Community Call and AMA

Waku will host a Community Call on X on the last Friday of each month. The objective of the call is to update the community on our progress and provide a space where questions can be put to the team 

The first Community Call took place last week on the 26th of January. You can [listen back to the full Space](https://twitter.com/Waku_org/status/1750927368644919722), and read the highlights below. 

[Vaclav](https://twitter.com/vpavlin), [Guru](https://twitter.com/hackyguru), and [Aaron](https://twitter.com/vrycmfy) from the Waku team participated in the call. 

### Waku network updates 

> **Vaclav**: If you don't know what the Waku Network is, it is our attempt to formalise a network of peer-to-peer nodes that would form some network running Waku protocols. We are very proud to have released the Gen 0, and there will likely be more generations to come right before Devconnect in Istanbul this November.

Reviewer: I think you meant that we released gen 0 before devconnect? The sentence reads weird as it seems to say that more gen will come right before devconnect.

Author: Updated

> It's just the beginning for the Waku Network; we're still dogfooding it. You can see [the list of milestones and the roadmap for improvements](https://github.com/waku-org/pm/issues/101) put forward by the research team. There’s also an interesting discussion you can join in the [Vac Forum](https://forum.vac.dev/t/message-validation-marketplace-waku-network-of-networks/262/2).

Reviewer: I think instead you can say that we are drafting the roadmap to come for 2024 (subject to change): https://github.com/waku-org/pm/blob/master/ROADMAP.md the post gen 0 issue was more a discussion artifact.

Author: updated

> As part of Generation 0 of Waku Network, we have integrated sharding, ensuring that it’s scalable to thousands of users and that the bandwidth is reasonable for people running on Raspberry Pis or smaller devices with weaker connectivity. Our client targeted at low-resource devices, JSWaku, has been making progress towards fully supporting sharding and, by that, supporting the full Waku Network.

Reviewer: Always mention rLN :)

I'd say "browser client" for js-waku to read more easily and be more accurate.

### Future incentivisation for running Waku nodes?

> **Vaclav: **We are often asked about how we incentivise node runners, as well as who is running the nodes and why. There are currently no monetary or token incentives. However, the research team has recently published findings that offer a glimpse into an incentivisation proposal. That's very interesting and promising for the future of Waku node operators to maybe get something in return.

Reviewer: I'd say instead that our researchers have defined the problem: https://github.com/waku-org/research/blob/master/docs/incentivization.md and are currently building a first PoC

### Growth of the Waku dev ecosystem

**Guru: **We had our first Waku hackathon at ETHRome last year where 120 hackers participated. We received eight project submissions from EthRome, which was a high number for us as a first-time sponsor. At ETHIstanbul our submissions increased to 20. We closed the year with ETHIndia where we had a huge total of 51 project submissions.

The graph has been trending consistently upwards as more hackers and developers discover Waku. We are exploring ways to offer support to these projects in terms of development, as well as raising funding and grants.

### Project integrations with Waku

> **Guru: **Waku has been integrated into several existing projects already. [Status](https://status.app/), of course, is the original use case for Waku, and they have been the first to benefit from the Waku Network. 

Reviewer: wellll. Status does not really the waku network. We can say that we are in the finali stages of collaborating with Status to integrate Waku's latest developments.

Author: updated

> [Railgun](https://www.railgun.org/) has been friends of Waku for a long period of time, and they integrated Waku around a year and a half ago. We also completed an integration with [The Grap](https://thegraph.com/)h, which uses Waku in its Graphcast SDK to decentralise its indexes.

Reviewer: ```suggestion
[Railgun](https://www.railgun.org/) has been friends of Waku for a long period of time, and they integrated Waku around a year and a half ago. We also completed an integration with [The Graph](https://thegraph.com/), which uses Waku in its Graphcast SDK to decentralise its indexes.
```

> [Silence Laboratories](https://www.silencelaboratories.com/) has also been researching building a multiparty computation (MPC) on top of Waku.

Reviewer: ```suggestion
[Silence Laboratories](https://www.silencelaboratories.com/) has also been building a multiparty computation (MPC) on top of Waku.
```

> [Tribes](https://www.tribes.xyz/) is also exploring how Waku can be implemented in their stack. Tribes are focused on privacy and censorship assistance, so it would be a great fit.

Reviewer: Are they cool for us to mention them?

Author: Vaclav is checking with the Tribes team before we publish
